devel/haskell-platform/Portfile

```diff
@@ -264,9 +264,11 @@ haskell_platform_port HUnit 1.2.5.2 3 {
         tool for Java, see: http://www.junit.org.
 }
 
-haskell_platform_port mtl 2.1.3.1 1 {
-    checksums               rmd160  f6432fb7e64d66eff69f02ccca9fc30ac5dce502 \
-                            sha256  77af766957fb49361fe63446f272a698bddb13398669edc363022e5c2517f6f6
+haskell_platform_port mtl 2.2.1 1 {
```

Comment:
Actually, for haskell ports it doesn't matter, because they can in fact be installed side-by-side. The keyword is `conflicts`, but you don't have to add it for hs-mtl.